CHANGELOG.txt

```diff
@@ -24,6 +24,23 @@ v23.0.00
 
     Significant Changes
         System: added ability to run CLI scripts remotely, with secure key
+        System: removed deprecated functions no longer used in core and additional modules
```

Comment:
Hmmm. I wonder if this list can be collapsed down, or perhaps the change is mentioned here and  list itself is at the bottom in deprecations? It takes up a lot of space 🤔

Reply:
My thinking was that having it take up space would be useful for anyone with custom code, as it'll drawn their attention to it.

Reply:
Fair enough.